libs/SalesforceAnalytics/SalesforceAnalyticsTests/InstrumentationEventBuilderTests.m

```diff
@@ -151,41 +158,37 @@ - (void) testAutoPopulateStartTime {
  * Test for auto population of mandatory field 'event ID'.
  */
 - (void) testAutoPopulateEventId {
-    SFSDKInstrumentationEvent *event = [SFSDKInstrumentationEventBuilder buildEventWithBuilderBlock:^(SFSDKInstrumentationEventBuilder *builder) {
-        double curTime = 1000 * [[NSDate date] timeIntervalSince1970];
-        NSString *eventName = [NSString stringWithFormat:kTestEventName, curTime];
-        builder.name = eventName;
-        builder.page = [[NSDictionary alloc] init];
-        builder.startTime = curTime;
-        builder.sessionId = kTestSessionId;
-        builder.senderId = kTestSenderId;
-        builder.schemaType = SchemaTypeError;
-        builder.eventType = EventTypeSystem;
-        builder.errorType = ErrorTypeWarn;
-    } analyticsManager:self.analyticsManager];
+    SFSDKInstrumentationEvent *event = [self standardTestEvent];
     XCTAssertTrue(event.eventId != nil, @"Event ID should have been auto populated");
 }
 
 /**
  * Test for auto population of mandatory field 'sequence ID'.
  */
 - (void) testAutoPopulateSequenceId {
-    SFSDKInstrumentationEvent *event = [SFSDKInstrumentationEventBuilder buildEventWithBuilderBlock:^(SFSDKInstrumentationEventBuilder *builder) {
+    SFSDKInstrumentationEvent *event = [self standardTestEvent];
+    NSInteger sequenceId = event.sequenceId;
+    XCTAssertTrue(sequenceId > 0, @"Sequence ID should have been auto populated");
+    NSInteger globalSequenceId = self.analyticsManager.globalSequenceId;
+    XCTAssertEqual(0, globalSequenceId - sequenceId);
+}
+
+#pragma mark - Helper methods
+
+- (SFSDKInstrumentationEvent *)standardTestEvent {
+    return [SFSDKInstrumentationEventBuilder buildEventWithBuilderBlock:^(SFSDKInstrumentationEventBuilder *builder) {
         double curTime = 1000 * [[NSDate date] timeIntervalSince1970];
         NSString *eventName = [NSString stringWithFormat:kTestEventName, curTime];
         builder.name = eventName;
         builder.page = [[NSDictionary alloc] init];
+        builder.attributes = @{ @"Name1": @"Value1" };
```

Comment:
Ah, I was adding something for `attributes` just to have it populated, testing some copy scenarios.  This doesn't need to be here.  I'll remove it.